code/extensions/Lumberjack.php

```diff
@@ -37,10 +37,20 @@ public function getExcludedSiteTreeClassNames() {
 	public function updateCMSFields(FieldList $fields) {
 		$excluded = $this->owner->getExcludedSiteTreeClassNames();
 		if(!empty($excluded)) {
-			$pages = SiteTree::get()->filter(array(
-				'ParentID' => $this->owner->ID,
-				'ClassName' => $excluded
-			));
+
+			// if we only have one $excluded class, use it instead of SiteTree
+			if (count($excluded) == 1) {
+				$pages = DataObject::get(array_keys($excluded)[0])->filter(array(

```

Comment:
Whilst not technically deprecated, DataObject::get() is the old 2.4 way of getting results. DataList::create($excluded[0]) is the new way of doing the same thing. That or $excluded[0]::get() but I think DataList works better with variables :)